### DISCUSSION TITLE
New Controls - DatePicker, MonthCalendar, Animation, etc?

### DISCUSSION THREAD

**Discussion by shqipstar** [OpenSourcePFCLibraries/2019#30] [Q&A] [answered]
Dear all, how do you integrate new controls, like DatePicker, MonthCalendar, Animation? Simply add into pfcmain and pfemain, as well as in the places where a case query makes the type distinction, or is there a specific procedure for this?

TIA,
gzim

**Comment by TABUllrich** [comment, marked as answer]
Yes, simple add the new controls in pfcmain and pfemain and look where they are missing in other functions. I 've already don eit e.g. with new webbrowser control.
I will now also add the three controls.
Do you miss anything else?

**Reply by shqipstar** [reply]
Very kind of you, thx!
